app/src/main/java/com/codesoom/assignment/ToDoHttpHandler.java

```diff
@@ -0,0 +1,174 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.models.Task;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.*;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+/**
+ * HTTP exchanges를 통해 전달받은 Request를 분석해서 할일 목록을 관리하고 적절한 Response를 전달하는 객체
+ */
+public class ToDoHttpHandler implements HttpHandler {
+    private ObjectMapper objectMapper = new ObjectMapper();
+    private List<Task> tasks = new ArrayList<>();
+    private Long lastId = 1L;
+
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+        final String method = exchange.getRequestMethod();
+        if (method == null) {
+            exchange.sendResponseHeaders(400, 0);
+            return;
+        }
+
+        final URI uri = exchange.getRequestURI();
+        final String path = uri.getPath();
+        if (path == null) {
+            exchange.sendResponseHeaders(400, 0);
+            return;
+        }
+
+        System.out.println(method + " " + path);
+
+        final String body = getRequestBody(exchange).orElse("");
+        if (!body.isBlank()) {
+            System.out.println(body);
+        }
+
+        if (method.equals("GET") && path.equals("/tasks")) {
```

Comment:
```suggestion
       "GET".equals(method)
```
요렇게 뒤집는 것만으로 더 안전한 코드를 만들 수 있어요!